src/checks/y_check_boolean_input_param.clas.abap

```diff
@@ -22,35 +25,12 @@ CLASS y_check_boolean_input_param IMPLEMENTATION.
     settings-threshold = 0.
     settings-documentation = |{ c_docs_path-checks }boolean-input-parameter.md|.
 
+    relevant_statement_types = VALUE #( ( scan_struc_stmnt_type-class_definition ) ).
+    relevant_structure_types = VALUE #( ).
```

Comment:
initial?

Reply:
it is not as the y_check_base has a default value